### ISSUE TITLE
No shared framework schemes

### ISSUE THREAD

**Issue by lunalugo25** [SDWebImage/SDWebImageSwiftUI#4]
Hi,

I tried to install this pod by Carthage and it fails when trying to compile the project. It says something like:
```
Dependency "SDWebImageSwiftUI" has no shared framework schemes
```

My solution:
- Open Xcode -> Manage Schemas
- Tap on "Autocreate schemes Now"
- Run in console $ carthage update

![Screen Shot 2019-09-24 at 9 37 42 AM](https://user-images.githubusercontent.com/18487961/65521710-45035480-deaf-11e9-8ab4-8ffb19a24ad0.png)

Xcode 11.0
Build version 11A420a

**Comment by dreampiggy** [comment]
Seems this version does not support Carthage (though it can be). The main part of this is to support the SwiftPM using Xcode project, which does not need **Shared Scheme** Config.

**Comment by dreampiggy** [comment]
Fixed by #5. You can have a try again.

Released [v0.1.1](https://github.com/SDWebImage/SDWebImageSwiftUI/releases/tag/0.1.1)
